### PR TITLE
feat(Adding --runonce flag so the tool would run watch once.):

### PR DESCRIPTION
**Pull request by jonycheung**
#40

Fixes #

- [x] Semantic naming
- [x] Clear and concise comments
- [x] Adequate tests

### CODE REVIEW

**Comment by eugene-matvejev** [comment]
does this means it will compile it at least once? e.g. if file doesn't exists it will create it?

**Comment by jonycheung** [comment]
@eugene-matvejev Yes and return to command prompt after the first run.

File: src/less-watch-compiler.js [hunk 1]
```diff
@@ -93,7 +95,10 @@ function init(){
       });
   }
   
-  console.log('Watching directory for file changes.');
+  if (lessWatchCompilerUtils.config.runonce === true)
+    console.log('Running less-watch-compiler once.');
```
Comment (eugene-matvejev) [inline src/less-watch-compiler.js]: consider wrap if/else block in {} brackets

Comment (jonycheung) [reply src/less-watch-compiler.js]: - [ ] Wrap if/else in {}

**Comment by eugene-matvejev** [comment]
do you have any idea when it will be merged? :)

I want get rid of legacy dependency in my spare-time project :)

**Comment by jonycheung** [comment]
Sorry. I've been away. Merging now..

**Pull request merged**